model_prices_and_context_window.json

```diff
@@ -6499,109 +6445,6 @@
         "mode": "chat" 
     },
-    "perplexity/pplx-7b-chat": { 
-        "max_tokens": 8192,
-        "max_input_tokens": 8192,
-        "max_output_tokens": 8192,
-        "input_cost_per_token": 0.00000007, 
-        "output_cost_per_token": 0.00000028, 
-        "litellm_provider": "perplexity", 
-        "mode": "chat" 
-    },
-    "perplexity/pplx-70b-chat": {  
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "max_output_tokens": 4096,
-        "input_cost_per_token": 0.00000070, 
-        "output_cost_per_token": 0.00000280, 
-        "litellm_provider": "perplexity", 
-        "mode": "chat" 
-    },
-    "perplexity/pplx-7b-online": { 
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "max_output_tokens": 4096,
-        "input_cost_per_token": 0.0000000, 
-        "output_cost_per_token": 0.00000028, 
-        "input_cost_per_request": 0.005,
-        "litellm_provider": "perplexity", 
-        "mode": "chat" 
-    },
-    "perplexity/pplx-70b-online": { 
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "max_output_tokens": 4096, 
-        "input_cost_per_token": 0.0000000, 
-        "output_cost_per_token": 0.00000280, 
-        "input_cost_per_request": 0.005,
-        "litellm_provider": "perplexity", 
-        "mode": "chat" 
-    },
-    "perplexity/llama-2-70b-chat": { 
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "max_output_tokens": 4096, 
-        "input_cost_per_token": 0.00000070, 
-        "output_cost_per_token": 0.00000280,
-        "litellm_provider": "perplexity", 
-        "mode": "chat" 
-    },
-    "perplexity/mistral-7b-instruct": { 
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "max_output_tokens": 4096, 
-        "input_cost_per_token": 0.00000007,
-        "output_cost_per_token": 0.00000028,
-        "litellm_provider": "perplexity", 
-        "mode": "chat" 
-    },
-    "perplexity/mixtral-8x7b-instruct": {
-        "max_tokens": 4096,
-        "max_input_tokens": 4096,
-        "max_output_tokens": 4096,
-        "input_cost_per_token": 0.00000007,
-        "output_cost_per_token": 0.00000028,
-        "litellm_provider": "perplexity",
-        "mode": "chat"
-    },
-    "perplexity/sonar-small-chat": {
-        "max_tokens": 16384,
-        "max_input_tokens": 16384,
-        "max_output_tokens": 16384,
-        "input_cost_per_token": 0.00000007,
-        "output_cost_per_token": 0.00000028,
-        "litellm_provider": "perplexity",
-        "mode": "chat"
-    },
-    "perplexity/sonar-small-online": {
-        "max_tokens": 12000,
-        "max_input_tokens": 12000,
-        "max_output_tokens": 12000,
-        "input_cost_per_token": 0,
-        "output_cost_per_token": 0.00000028,
-        "input_cost_per_request": 0.005,
-        "litellm_provider": "perplexity",
-        "mode": "chat"
-    },
-    "perplexity/sonar-medium-chat": {
```

Comment:
this pr removes pricing. but i see a few new perplexity models - can those be added within this pr? @abromberg

Reply:
Unless I'm misunderstanding something, my updates bring it in line with the [currently supported models](https://docs.perplexity.ai/guides/model-cards):

<img width="811" alt="Xnapper-2024-12-08-15 42 31" src="https://github.com/user-attachments/assets/c64f9901-bf64-4565-9f8a-6b60624d7746">

Those are the three models I left in the pricing files. Is there something else you want me to do?

(I also just added the per-request cost in commit edaa030 on this PR)